CHANGELOG.org

```diff
@@ -0,0 +1,11 @@
+see also the changelogs of ~smtlib-backends-tests~, ~smtlib-backends-process~ and
+~smtlib-backends-z3~
+
+* v0.2
+** split the ~Process~ module into its own library
+** rename ~SMTLIB.Backends~'s ~ackCommand~ to ~command_~
+** remove logging abilities
+the user can always surround ~command~ or ~command_~ with their own logging
+functions
+** improve read-me
```

Comment:
Is this some standard change log format?

For reference, here's how other repos do it:
https://github.com/tweag/capability/blob/master/ChangeLog.md
https://github.com/tweag/inline-java/blob/master/CHANGELOG.md
https://github.com/tweag/linear-base/blob/master/CHANGELOG.md

Reply:
It's just Emacs `org-mode` format, which I used because I like Emacs and wanted to nest list items arbitrarily. But it seems you can do the same in Markdown so I'll switch to Markdown.